components/widgets/menu/menu_items/menu_item_action.tsx

```diff
@@ -42,23 +42,7 @@ export const MenuItemActionImpl = ({
         }
         onClick={onClick}
     >
-        {text && (
-            <React.Fragment>
-                <span>
-                    <span className='MenuItem__primary-text'>{text}</span>
-                    <span
-                        className={classNames([
-                            'MenuItem__right-decorator',
-                            {
-                                'MenuItem__text-color': !isDangerous,
-                            },
-                        ])}
-                    >
-                        {rightDecorator}
-                    </span>
-                </span>
-            </React.Fragment>
-        )}
+        {text && <span className='MenuItem__primary-text'>{text}{rightDecorator}</span>}
```

Comment:
```suggestion
        {(text && rightDecorator) && <span className='MenuItem__primary-text'>{text}{rightDecorator}</span>}
```

Reply:
<img width="312" alt="image" src="https://user-images.githubusercontent.com/7575921/160288008-9a800ec8-ac7f-4a3a-aa47-8bd6e49462dc.png">


<img width="341" alt="image" src="https://user-images.githubusercontent.com/7575921/160288024-ac3bb09e-3f3b-4f41-873e-d102cf75f9e7.png">

Reply:
@michelengelen I'm not sure I agree with this one because the right decorator is an optional argument to the component. checking if it exists to render the menu item will omit menu items without rightdecorator.

Reply:
oh yes ... you are right.